webcompat/static/css/src/issue.css

```diff
@@ -60,9 +70,33 @@
   justify-content: center;
 }
 
+.comment-body {
+  max-width: 100%;
+  overflow: auto;
+}
+
 /* As we can not add a class to the img right now (markup parsers are complicated), we need to use the element */
 .comment-body img,
 .js-Issue-markdown img {
   max-width: calc(var(--unit-modul-m) - (var(--unit-space) * 4));
```

Comment:
One last detail. The max-width if the image is 550px. The breaking point 650px. The image parent box `comment-body` gets an overflow-x, which maybe is fixable?

If this is an issue, we can also file a follow up. Just want to make sure, we have everything set as there will be work build upon that change.